packages/mdc-tabs/README.md

```diff
@@ -637,6 +637,10 @@ Scrolls the tab bar such that the rightmost tab traverses the scroll frame and b
 
 > **NOTE:** Due to a quirk in event behavior, we allow the rightmost tab to be partially occluded even when tabbed to because clicking on such an element would shift the frame on the `focus` event. This would result in a scenario where the ripple persists and the intended tab would not be selected due to the tab bar shifting before the `mouseup` or `click` events get dispatched.
 
+#### MDCTabBarScrollerFoundation.scrollToTabAtIndex(index) => void
```

Comment:
Sorry for not noticing this at the same time as my other comment here, but we need to specify the type of the parameter here: `(index: number)` (I guess that might've gotten mixed up into the return value earlier).

Reply:
Thanks, good catch. I have updated this as well.